lib/HTML/FormBuilder/Select.pm

```diff
@@ -63,8 +65,12 @@ has name => (
 has options => (
     is  => 'rw',
     isa => sub {
-        die "$_[0] is not ArrayRef[HashRef[Any]]"
-            unless (ref($_[0]) eq 'ARRAY' && ref($_[0][0]) eq 'HASH');
+        my $arg = $_[0];
+        my $msg = Dumper($arg) . " is not ArrayRef[Maybe[HashRef]]";
+        die $msg if !ref($arg) || ref($arg) ne 'ARRAY';
+        for my $elem (@$arg) {
+            die $msg if (!ref($elem) || ref($elem) ne 'HASH');
+        }
```

Comment:
I didn't get it. In this case, it still dies if caller fail to pass in "options" as arrayref of hashref. What I mean previously was, this piece of code shouldn't even die at all, just accept empty options silently